src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -254,28 +246,16 @@ private static void register(int id, Set<String> models, BiConsumer<Position, By
         register(200, fmbXXX, (p, b) -> p.set("sleepMode", b.readUnsignedByte()));
         register(205, fmbXXX, (p, b) -> p.set("cid2g", b.readUnsignedShort()));
         register(206, fmbXXX, (p, b) -> p.set("lac", b.readUnsignedShort()));
-        register(232, fmbXXX, (p, b) -> p.set("cngStatus", b.readUnsignedByte() > 0));
-        register(233, fmbXXX, (p, b) -> p.set("cngUsed", b.readUnsignedInt() * 0.1));
-        register(234, fmbXXX, (p, b) -> p.set("cngLevel", b.readUnsignedShort()));
-        register(235, fmbXXX, (p, b) -> p.set("oilLevel", b.readUnsignedByte()));
+        register(232, fmbXXX, (p, b) -> p.set("cngStatus", b.readUnsignedByte() > 0)); //only available via additional CAN reader
+        register(233, fmbXXX, (p, b) -> p.set("cngUsed", b.readUnsignedInt() * 0.1)); //only available via additional CAN reader
+        register(234, fmbXXX, (p, b) -> p.set("cngLevel", b.readUnsignedShort())); //only available via additional CAN reader
+        register(235, fmbXXX, (p, b) -> p.set("oilLevel", b.readUnsignedByte())); //only available via additional CAN reader
```

Comment:
Why do we need these comments?

Reply:
It is absolutely expected that not all of the parameters will be reported by all devices. That's fine as long as there are no conflicts. Please remove these comments.

Reply:
Commenting removed. Still not comfortable with replacing the original AVL ids with custom names. Equally, someone might prefer to call it gasStatus or lpgStatus.